README.md

```diff
@@ -1,13 +1,36 @@
 # Chainblocks examples
 
-Example games and tutorials using [chainblocks](https://github.com/fragcolor-xyz/chainblocks).
+[![license](https://img.shields.io/github/license/fragcolor-xyz/chainblocks-examples)](./LICENSE)
+[![docs](https://img.shields.io/badge/docs-API-blueviolet)](https://docs.fragcolor.xyz/)
```

Comment:
I would add a shield for the "learn" website that is built from this repository. Same as the one in the `chainblocks` repo.

```suggestion
[![docs](https://img.shields.io/badge/docs-API-blueviolet)](https://docs.fragcolor.xyz/)
[![examples](https://img.shields.io/badge/learn-examples-blue)](https://learn.fragcolor.xyz/)
```

Reply:
Fixed